bin/utils.js

```diff
@@ -233,12 +231,18 @@ exports.setupWSConnection = (conn, req, { docName = req.url.slice(1).split('?')[
         conn.ping()
       } catch (e) {
         closeConn(doc, conn)
+        clearInterval(pingInterval)
```

Comment:
Right, good catch!

Because this interval handler is still active for every failed connection, we keep all of the above variables in scope (e.g. doc, conn, req, ...).